server/routes/index.js

```diff
@@ -1,5 +1,5 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 // const { quarterlyStockInfo } = require('../dal/stock-markets');
```

Comment:
Would be nice to remove these comments

Reply:
wasn't this the line that was giving the error when Raghav uncommented?
anyway I can delete it if you want